src/sentry/conf/server.py

```diff
@@ -1132,7 +1139,7 @@ def create_partitioned_queues(name):
 )
 
 # Delay (in ms) to induce on API responses
-SENTRY_API_RESPONSE_DELAY = 0
+SENTRY_API_RESPONSE_DELAY = 150 if IS_DEV else None
```

Comment:
@mattrobenolt devserver could be a good approach. I'm not sure if theres any real negatives (e.g. areas we'd need it and not have it)?